### ISSUE TITLE
Issue with notation in book

### ISSUE THREAD

**Issue by Hamish-Gibson** [mfrdixon/ML_Finance_Codes#5]
I'd like to start by saying that this isn't an issue with any of the code, rather an issue with the notation in the book. I didn't know where best to raise the issue directly so I thought this would be the most appropriate place. 

I recently purchased this book and have been thoroughly enjoying it so far, I am currently on Chapter 6: Sequence Modelling. On page 194, section 2.2 _Autoregressive Processes_ where it discusses the backshift operator and I came across notation for which I thought was confusing as it is written differently to my studies. 

Just above equation 6.3, it describes the concept of the backshift operator as https://render.githubusercontent.com/render/math?math=y_{t-j}=L^{j}[y_{j}] 
But surely this is wrong? As surely applying a backshift operation j and timestamp j, would just yield y0? And not y_t? 

Wouldn't the correct notation be
https://render.githubusercontent.com/render/math?math=y_{t-j}=L^{j}[y_{t}]  
Because yielding y_{t - j} would require the backshift operator of order j to be applied at y_t?

I apologise if I am wrong for this, but it just seems like something is off here.

**Comment by mfrdixon** [comment]
Yes you are correct, there is a typo. it should be y_t inside the square
parentheses. Thanks for pointing this out.
